mocks/Client.go

```diff
@@ -513,3 +513,17 @@ func (_m *Client) TerminateWorkflow(ctx context.Context, workflowID string, runI
 
 	return r0
 }
+
```

Comment:
Fixed mocks manually but I think they can be safely removed. These are generated with some 'mockery` framework and we use `gomock` everywhere else. I tried to use `mockgen` to generate mocks and it generate correct mocks. So I would allow customers to generate mocks themselves.

Reply:
#61